frontend/src/styles.scss

```diff
@@ -246,3 +246,21 @@ body {
 .ant-notification-notice-message {
 	padding-right: 20px;
 }
+
+.ant-drawer {
+	// do not modify this as this is required to hide intercom in modals
+	z-index: 1000000000000;
+}
+
+.ant-modal-root {
+	.ant-modal-mask {
+		z-index: 2000000000000 !important;
+	}
+	.ant-modal-wrap {
+		z-index: 2000000000000 !important;
+	}
+}
```

Comment:
please make sure to test these style specifically as these will be globally applied

Reply:
it's tested but let me know if you find any issues in testing with the same